hazelcast/src/main/java/com/hazelcast/internal/serialization/SerializationServiceBuilder.java

```diff
@@ -63,5 +64,7 @@
 
     SerializationServiceBuilder setInitialOutputBufferSize(int initialOutputBufferSize);
 
+    SerializationServiceBuilder setClusterService(ClusterService clusterService);
```

Comment:
To reduce coupling, the Cluster version (String?) could be used instead of ClusterService.